charts/bitwardenrs/README.md

```diff
@@ -0,0 +1,48 @@
+# Unofficial Bitwarden compatible server written in Rust
+
+This is an opinionated helm chart for [bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) 
+
+The default values and container images used in this chart will allow for running in a multi-arch cluster (amd64, arm, arm64)
+
+## TL;DR
+
+```console
+helm repo add billimek https://billimek.com/billimek-charts/
+helm install billimek/bitwarden_rs
```

Comment:
Need to update these names, bitwardenrs, right?

Reply:
fixed